Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM php:5.6
+
+RUN echo "date.timezone = UTC" > /usr/local/etc/php/conf.d/date.ini
```

Comment:
I suppose that `libicu-dev` needs for `intl`, right? But why we need `g++` here?